mediaire_toolbox/transaction_db/migrations.py

```diff
@@ -90,6 +90,25 @@
         "  FOREIGN KEY (user_id) REFERENCES users(id),"
         "  FOREIGN KEY (site_id) REFERENCES sites(id)"
         ");"
+    ],
+    19: [
+        "ALTER TABLE transactions ADD COLUMN patient_consent_date DATETIME DEFAULT NULL;",
+        ("UPDATE transactions SET"
+         "  patient_consent_date = data_uploaded "
```

Comment:
@phistep maybe creation_data is a better default? as data_uploaded we know has quite some delay sometimes, and might be null for several studies that we couldn't upload

Reply:
sounds reasonable